torchvision/csrc/cpu/image/readjpeg_cpu.cpp

```diff
@@ -1,18 +1,16 @@
 #include "readjpeg_cpu.h"
 
 #include <ATen/ATen.h>
-#include <setjmp.h>
 #include <string>
 
 #if !JPEG_FOUND
-
-torch::Tensor decodeJPEG(const torch::Tensor& data) {
+torch::Tensor decodeJPEG(const torch::Tensor& data, int64_t components) {
```

Comment:
Components or Channels? All JPEG implementations seem to use the word components instead of channels. On the other hand PNG implementations use the word channels.

I ended up using the most accurate word in each implementation but this makes us use the 2 words interchangeably across the codebase, which is definitely not ideal.

Reply:
done